practice-two/src/components/common/Button.jsx

```diff
@@ -9,6 +9,7 @@ const variantClasses = {
   primary: `${BUTTON.DEFAULT} rounded-lg`,
   secondary: `${BUTTON.DEFAULT} rounded-[50px]`,
   outline: 'border-none',
+  custom: `${BUTTON.DEFAULT} py-2 px-4 w-[100%] lg-[50%] rounded text-white text-xl font-semibold justify-center`,
```

Comment:
does this one is difference from primary or secondary ?

Reply:
Updated